### ISSUE TITLE
Error in Firestore Windows auth

### ISSUE THREAD

**Issue by Bilonik** [FlutterWay/firebase_for_all#1]
First of all thank you very much for this package.

This is an error of unauthorized when firestore is set with allow read, write: if request.auth != null;.

Works on ios and Android but not in Windows.

**Comment by Bilonik** [comment]
@worldwidee  I think you are not passing auth to the windows firedart:

You can also instantiate your own Firestore object. Please note that if your database requires authenticated access, you'll need to pass along an instance of FirebaseAuth.

var firebaseAuth = FirebaseAuth.(apiKey, await HiveStore());
var firestore = Firestore(projectId, auth: firebaseAuth);

**Comment by worldwidee** [comment]
Yes, I didn't know there was such a requirement in the firedart library. Unfortunately, adding them is not enough. Because firedart does not listen to the original FirebaseAuth library. It needs to be initialized() with its own FirebaseAuth and when signIn is done, firedart's auth library also needs to be signed in.
Working on it
